readme.md

```diff
@@ -184,7 +169,209 @@ Working directory from which to resolve a `contributors` module, if any
+
+```markdown
+# Hello
+
+Some text.
+
+## Contributors
+```
+
+You can now use the CLI to format `example.md`:
+
+```sh
+./node_modules/.bin/remark --output --use remark-gfm --use remark-git-contributors example.md
+```
+
+This adds the table of contributors to `example.md`, which now contains (when
+running in this project):
+
+```markdown
+# Hello
+
+Some text.
+
+## Contributors
+
+| Name                | GitHub                                       | Social                                                |
+| :------------------ | :------------------------------------------- | :---------------------------------------------------- |
+| **Vincent Weevers** | [**@vweevers**](https://github.com/vweevers) | [**@vweevers@twitter**](https://twitter.com/vweevers) |
+| **Titus Wormer**    | [**@wooorm**](https://github.com/wooorm)     | [**@wooorm@twitter**](https://twitter.com/wooorm)     |
+```
+
+### Example: CLI in npm scripts
+
+You can use `remark-git-contributors` and [`remark-cli`][cli] in an npm script
+to format markdown in your project.
+Install both (and [`remark-gfm`][remark-gfm]) with [npm][]:
+
+```sh
+npm install remark-cli remark-gfm remark-git-contributors --save-dev
+```
+
+Then, add a format script and configuration to `package.json`:
+
+```js
+{
+  // …
+  "scripts": {
+    // …
+    "format": "remark . --quiet --output",
+    // …
+  },
+  "remarkConfig": {
+    "plugins": [
+      "remark-gfm",
+      "remark-git-contributors"
+    ]
+  },
+  // …
+}
+```
+
+> 💡 **Tip**: Add other tools such as prettier or ESLint to check and format
+> other files.
+>
+> 💡 **Tip**: Run `./node_modules/.bin/remark --help` for help with
+> `remark-cli`.
+
+Now you format markdown in your project with:
+
+```sh
+npm run format
+```
+
+### Example: `appendIfMissing`
+
+The default behavior of this plugin is to not generate lists of Git
+contributors if there is no `## Contributors` (case- and level-insensitive).
+You can change that by configuring the plugin with
+[`appendIfMissing: true`][append-if-missing].
+
+The reason for not generating contributors by default is that as we saw in the
+previous example (CLI in npm scripts) remark and this plugin often run on
+several files.
+You can choose where to add the list by explicitly adding `## Contributors`
+in the main file (`readme.md`) and other docs won’t be touched.
+Or, when you have many contributors, add a specific `contributors.md` file,
+with a primary `# Contributors` heading, and the list will be generated there.
+
+To turn `appendIfMissing` mode on, pass it like so on the API:
+
+```js
+  // …
+  .use(remarkGitContributors, {appendIfMissing: true})
+  // …
+```
+
+Or on the CLI (in `package.json`):
+
+```js
+  // …
+  "remarkConfig": {
+    "plugins": [
+      // …
+      [
+        "remark-git-contributors",
+        {"appendIfMissing": true}
+      ]
+    ]
+  },
+  // …
+```
+
+### Example: metadata
+
+The data gathered from Git is somewhat sparse.
```

Comment:
What do you mean by sparse in this context?

Reply:
only name and email, whereas this plugin supports URLs and socials as well

Reply:
```suggestion
The data gathered from Git is somewhat sparse, only name and email.
```